### ISSUE TITLE
Add support for TV series rename

### ISSUE THREAD

**Issue by Tarek-Hasan** [sun-jiao/flut-renamer#24]
Please, add support for TV/Web series rename like [TV Rename](https://github.com/TV-Rename/tvrename), which uses [The TVDB](https://thetvdb.com/) for data. Additionally, function to automatically sort to episode number so wrong title is not assigned.

I don't want to use a different renamer app, just for renaming TV series - a feature I may use once or twice a year. Also, TV Rename is a media management tool where I just want to simply rename episodes.

**Comment by sun-jiao** [comment]
Seems [The TVDB](https://thetvdb.com/) is an online database, right?

**Comment by Tarek-Hasan** [comment]
Yes, and here is the TVDB [API](https://thetvdb.com/api-information).

**Comment by sun-jiao** [comment]
I'm not currently considering adding any functionality that relies on the internet.
